### ISSUE TITLE
Can't define more that 3 types of terrain for dungeon

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#75]
I've defined

```
wall:dark wall 0:48
wall:dark wall 1 narrow window:1
wall:dark wall 2 window:1
wall:dark wall 3 candelabra:1
wall:dark wall 4 torch:1
wall:dark wall 5 lamp:1
wall:dark wall 6 niche:1
wall:dark wall 7 swords:1
.... and 50 more :)
```
(could be seen at https://tangaria.com/data/dungeons/ ; dungeon 'Abandoned Manor'.
; ctrl+f5 to refresh page)

but only three terrain works:
```
wall:dark wall 0:48
wall:dark wall 1 narrow window:1
wall:dark wall 2 window:1
```

others doesn't show up in the dungeon

I've tried to comment out different terrain, it didn't work; still only 3 shown up

**Comment by draconisPW** [comment]
Probably hardcoded. I'll check...

**Comment by draconisPW** [comment]
Definitely hardcoded: 5 rules, 3 floors, 3 walls. I'll remove this.

**Comment by draconisPW** [comment]
Implemented in ea0e3b4.
